x-pack/platform/plugins/shared/lens/public/visualizations/datatable/components/density_settings.test.tsx

```diff
@@ -31,9 +31,10 @@ describe('DensitySettings', () => {
 
   it('renders the density settings component with label', () => {
     renderDensitySettingsComponent();
-
-    expect(screen.getByLabelText('Density')).toBeInTheDocument();
     expect(screen.getByTestId('lnsDensitySettings')).toBeInTheDocument();
+    expect(
```

Comment:
It should be sufficient to add a `label` selector to the existing line to make it ignore the `aria-label` only check:

```suggestion
expect(screen.getByLabelText('Density', { selector: 'label' })).toBeInTheDocument();
```

Reply:
I think that because how the EUI components work, I think we must use `getByText` with the label selector since the `getByLabelText` searches the one that is related to the legend.